lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -147,3 +147,10 @@ bool CPUBackend::canDoIndexTypeDemotion(
       Kinded::Kind::SparseToDenseMaskNodeKind);
   return fromTy == ElemKind::Int64ITy && toTy == ElemKind::Int32ITy;
 }
+
+#if !FACEBOOK_INTERNAL
+#include "cpuObjectRegistry.h"
+llvm::ArrayRef<llvm::MemoryBufferRef> CPUBackend::getObjectRegistry() const {
```

Comment:
I think this function should always have an implementation, even under FB_INTERNAL as the class defines this method as existing. But the body of this method should return an empty arrayref and not use cpuObjectRegistry.

Or alternatively, instead of getting cpuObjectRegistry from the include file above, you can define it as a static empty array and return it here.

Reply:
You're right. What about now?